### ISSUE TITLE
Patchbot server do not get the right list of tickets

### ISSUE THREAD

**Issue by videlec** [robertwb/sage-patchbot#76]
It seems that the patchbot server sometimes forgot about certain tickets...

Monday 11th of January 2016, ticket [#16137](http://trac.sagemath.org/ticket/16137) but patchbot is constantly ignoring it since 2 days. I checked in my patchbot log (machine librae) and there is no line for 16137.

**Comment by fchapoton** [comment]
Maybe because of
"Matthieu Dien": "not trusted", ?

2016-01-11 23:45 GMT+01:00 videlec <notifications@github.com>:

**Comment by videlec** [comment]
You mean that the server is ignoring tickets with untrusted authors? I thought this was filtered on the client side. The client logs explicitly mentions when there are untrusted authors.

**Comment by fchapoton** [comment]
No, there is no filter at the server level, I think. So there may be a true problem indeed.

**Comment by videlec** [comment]
Other ignored ticket http://trac.sagemath.org/ticket/269

**Comment by dkrenn** [comment]
At the very moment, https://patchbot.sagemath.org/ticket/22392/ seems to be ignored.

**Comment by saraedum** [comment]
Is this #116?

**Comment by dkrenn** [comment]
> Is this #116?

Maybe.
